docs/examples/tutorial_peninsula_AvsCgrid.ipynb

```diff
@@ -0,0 +1,432 @@
7qPGEf8iPG4+Z4fwvHAqhwqzCpin05uJI5u3aYUQLN9q7ztPMB2HlRYV0BBvTSO9PBLkkVPHTh/8P6//6N+ZSqm6mrqly5GM3MWzsnJF+z8giDg08kNn05uJI3phkFvpORkNQXHKyk8UUVpVg2iyVL3qrKwHtV5eHb8U/j5kTtQGg3oGzVtOt7V24ewPgMumE2VaBLZ8H0a1cVS4MunsxvJM1r3uq5fucJyvadwWNYWf9FeMVD3o+Vc7h1K9PaEh6MnHo5Shr7b1GnU/PITuiNH0B07imHxH4y8ZTrLPtqLaBI5tDYXd19nug+zzOkdXd2Y8sRLbPr+S7kQ9aHVyyg8nsboux7Gr2vLdj7/ZlyaM7p2QtHxNJysfKBUDo50SexFRN9BhPbpj6NL+6oPW8PWP0/I/lEuXo4MvzH+tCScPjsL7QFpsu8QE4vDKaluB8r2yds2fugFFh85MeTinxi2F3r4J7H45EIADpTttyHRnYcMRXBxQWxooGHNasRnn0doJbDi4KRi5K0JLH5nN0aDiaNbCvDp7E7ckPMXAQbpHo4aMISoAUPkwl7VxQXnxb7kn4ZeU2/xcRMEAiOiCe83iIjklAtCvjUh52AZu5aYVdECDL+x+2kJG1EUqV+yWH7tOn68zf4DZfvl7R7+Vn3XmkT3kSYnVQ0WOxdv54s34ttWiD4REoleVwzaWnB0J8Hq+WYdPARQuLvjMmSoZOlSVkbj7l04J/c/9W1tMGBWFCVZ1VQW1FNV1MCG79MY/Z8ep30mnwsEQSA4pjvBMd0ZcuOd5KcdIGP7ZsJ6t36tFyPqykosY68gEBwVR0RyCuH9BuLud+EKBlUV17P6qwNy8aPYwZ2IP03wWtTpaDDbMAlmy5FTYTQZ5QVAiGsnPM2TU1ok0SUizkGlQHke77F/FO5BiIISQTQiVFsyD6x90Y9UHGZo5+Hya4WTE66jx1C3cAFiXR2ajRuaLbROhSAIDLkulrrKRkpOVtNQrWPF5/uZ/GCfswqEOXt4MvyWe+gxehJbfv6W3EN7AakOyYbvPmPj91/QKa4HEckphPUZgKt3+3g5/5sR0W+QTKJ7BAQRkZxCRPIgAsKjL5g6v66ikZWf78dg7jvhfQLoPen0C7W6ZUvlbbcWAmDWc+ckf/M80GSEWolAtiaVLzdP9KbvACDKy1LU+WDZfkZ3G9esrcvIkQhOToiNjdSvSMX3yadaLEro4KRi7J1J/PX2LjS1Ooozqtn405FWa1O0BxycXWRCXd/YSPb+XTKhrm/UENHPflHjiw0nd/5tU3Ojc3wS8aMmENZnwHm1raksrGdvqpRFKQgw+JrYVm1Km9BoVVTUacAAm33WVi69AzrEYBcCSi9vvB94iPIXnweg/JWXCPn1dxSuF44rsYZKrSQkxoeQGGnM1WsNFGc0keqVVBbUExB26ax9G6oqbDirU+Hs4UlgZAwhsQl0ju+JX7ewC5ott2NhOjmHpAC1g4uKUbcmoGols1I0Gi3Waio1LiNHWvaJllobAc4BeDl5n1LYu4NEP18QlEp8n32ewmuvluybPvmYTmPGMGhONFt+k36TrX8ex9FVRURfy++gVKkZfuu9BEREs/F/X2DU6yjPyWLuMw/SZ9psek+ZfdGp0jtI9FagdnbF08+Prom96ZrYh87xif+I9+zh9bly+pFCJTDyloQ2eXnVLbUsBuyp4Q6USkScUlDaeH4rrEj0jqKiFvSwIeL22+xTODnhMnIU9UsWY6qpoWHTRlxHtW6P4tvZnZSrY9j4o2Sds3Xucdx9neTqxucbjq5uxA69+D2pWoKDqzuRvfvRNbE33Xr2vSCK81NRnlfL+v8dlpVwvSeGten31aWloc+UFhVOffuiCratZn2gdJ+8nejXU94WKqT0NhHh8lSiAybvcBS5kkJJqMxEDEok3DMCZ5UzGoOmWd8FqTBSk6VL3ZLFpyXRVQ5KRt/eg0Vv70LXYCDnUBk7TklhO59QqlTmcan3BTnfhYZCqaRLQi/C+w38x4hHTY2OlZ/vR9sgkWCdYn0YODvqtIuOhg3rMdVKNmmuo0ajcG7uz55bm43GIKmFYnzi5P8L1RYbE2sirskSwuUSJeEAUKgkW4iaPBs7l0SrIOGW/I3ckXiPTTPXCZOoW7gAkAjQ05HoIC2uR9/eg8Xv7qK2rJHKgnrWfHOQsXcltVhs8nTw7RrKlCdeIufAHrb98YNcGFg0meRaBhv+9zlBkTGE9R1IeN+BeAWFnOZdL06EJw+isb6OiORB+HYJveC2NtoGPSs+24+mRhr7AsI9GXJd3Gmvw9TYSL25iJng5oazlf+qNQ6WWgKx8rywrghBlPqp6GEJtNl4ol+i/dfGE92q7/YM6INSUGIUjazPW8sDvR9t9hsoXFxxGTGS+uXLMFVX07Bpk10LnSa4+Tgx5s5Eln24B4POxMndxbj5ONFvWkT7fzA7UDs5SbWE+g/GoNORe2gvIbEJp294EaBLYm9qcjKJHTaa+BHj8Ao+v6IeMNu4/HREDlQnjOraqt2S3K6hgcZ90lxO1bUr6hDba91dvFPe7h3YsY69UHCfOYu6+fPQHjyA/mQGRbfdQuAXX9otznyhoXZU0bm7r7wGM+iMqC6hZ7KLlzeent44urri4umNq48vngHBeAV3wqdLN9x8/P4xi7m0DXkcWiuJIwSFwMibE05bC65x926MJVKxUJfBg23uoaL6QtmOuEloIdRKJLqoUIPzpS9W+CfhlJiE+5wrqf39N0RNA2XPPE3MN/+ltryRA6uyQYQNPxxBqVYSmmRbd6j78LEERkSz6tN3KM/NwmQ0sHP+rxz/ewMp19xKaO/ki8YKsYNEbwVXvvohgZ1Pb5lyPnFydzHb5p2QXw++OpbA8NNHTkVRpG7JX9ILQcBt4iSb/bW6Gk5WS4RbtHcsTirLQl+Rby4q6uCO6HthSKGLAeGeEbiq3ajX13GwbL9NkTOQVEtN6uG6RQtPS6IDRPUPpiK/j
```

Comment:
"Parcels" -> "`Parcels`".

Reply:
I think its fine having "Parcels" without the code formatting?

Reply:
Keep as is for consistency with other tutorials. Micael agrees